scripts/bump.ts

```diff
@@ -0,0 +1,67 @@
+import * as fs from 'fs';
+import simpleGit, { SimpleGitOptions } from 'simple-git';
+
+// Increments the specified version based on the release type
```

Comment:
Prefer to use /** style comments for JSDoc compat.
Also applies to other definition comments.